recipes/Getting_Started_with_Granite_Code.ipynb

```diff
@@ -15,29 +15,39 @@
    "source": [
     "## Introduction\n",
     "\n",
-    "This notebook demonstrates using inference calls against a model hosted on [Replicate](https://replicate.com/). To see how you can use [Ollama](https://ollama.com/) to host models locally instead, see the [Coding Assistant in VSCode](Coding_Assistant_in_VSCode/Coding_Assistant_in_VSCode.ipynb) recipe."
+    "This notebook demonstrates using inference calls against a model hosted on [Replicate](https://replicate.com/).  To see how you can use [Ollama](https://ollama.com/) to host models locally instead, see the [Continue VSCode](Continue_VSCode/Continue_VSCode.ipynb) recipe.\n",
```

Comment:
We're planning to refactor out the Ollama instructions into a separate notebook, right? I definitely think we should.